src/protocol/notarization/commitment2.md

```diff
@@ -0,0 +1,16 @@
+# Commitment
+
+The TLSNotary protocol completely hides the plaintext transcript from the `Notary`. At the same time we want to have a way for the `User` to selectively prove the authenticity of arbitrary portions of the plaintext to the `Verifier`.
+
+Naively, we could extend the above `Encryption and Decryption` steps to also compute a commitment (e.g. a blake3 hash) to the plaintext in MPC and have the `Notary` sign that commitment. Then the `User` could open the commitment to the `Verifier`. Unfortunately such approach would be resource-heavy, and so we describe a more lightweight way of producing commitments. 
+
+The high-level idea is that the `User` will commit to the [encodings](/building_blocks/encodings.md) of the plaintext during the execution of the [DEAP](/building_blocks/deap_deferred.md) protocol. Since those encodings are chosen by the `Notary` and are not known to the `User` at the time when she makes a commitment, they can be thought of as "authenticated plaintext". 
```

Comment:
Do you think saying "we reuse the encodings from the MPC protocol used for `Encryption and Decryption` to create commitments (see here for details)" is high-level enough?

if we say "we reuse the commitments generated during the MPC" that wouldn't be accurate because our MPC doesn't require those commitments.  

Im also fine with saying "we reuse some internal values from the MPC protocol used for `Encryption and Decryption` to create commitments (see here for details)"

Reply:
@sinui0 ---^